docs/src/main/sphinx/admin/properties-general.rst

```diff
@@ -54,3 +54,23 @@ remove usage of this property.
 Ensure to use this only as a temporary measure to assist in your migration
 efforts.
 
+``protocol.v1.prepared-statement-compression.length-threshold``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+* **Type:** :ref:`prop-type-integer`
+* **Default value:** ``2048``
+
+Prepared statements that are submitted to Trino for processing, and are longer
+than the value of this property, are compressed for transport via the HTTP
+header to improve handling, and avoid failures due to hitting HTTP header size
+limits.
+
+``protocol.v1.prepared-statement-compression.min-gain``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+* **Type:** :ref:`prop-type-integer`
+* **Default value:** ``512``
+
+Prepared statement compression is not applied if size gain is less than the
+configured value. For these smaller statements no benefits occur from
```

Comment:
Not sure what "these" refers to. Shouldn't that be removed if there's no reference elsewhere to a certain set of statements?